pkg/cli/admin/prune/deployments/deployments.go

```diff
@@ -80,6 +81,7 @@ func NewCmdPruneDeployments(f kcmdutil.Factory, streams genericclioptions.IOStre
 
 	cmd.Flags().BoolVar(&o.Confirm, "confirm", o.Confirm, "If true, specify that deployment pruning should proceed. Defaults to false, displaying what would be deleted but not actually deleting anything.")
 	cmd.Flags().BoolVar(&o.Orphans, "orphans", o.Orphans, "If true, prune all deployments where the associated DeploymentConfig no longer exists, the status is complete or failed, and the replica size is 0.")
+	cmd.Flags().BoolVar(&o.ReplicaSets, "replica-sets", o.ReplicaSets, "If true, prune ReplicaSets associated with Deployments.")
```

Comment:
Let's mark this as `experimental` to send appropriate signal that it's not fully tested.